### ISSUE TITLE
Training Custom dataset

### ISSUE THREAD

**Issue by arkinrc** [Beniko95J/MLF-VO#2]
@Beniko95J 
Hello,
I would like to appreciate your efforts. I have successfully implemented pretrained model with the kitti dataset but when applied with my own dataset, there is huge difference in results. So i would like to know regarding the custom dataset training? do you have any plan to opensource the training file? my results are given below.

![combine_images](https://user-images.githubusercontent.com/42382784/180706884-1eb608a0-b518-4ae0-aec9-86f0f33b5a21.jpg)

looking forward to your feedback....
regards
arkin

**Comment by Beniko95J** [comment]
Sorry for the late reply. I have uploaded the files for training in the `dev` branch although they still looks messy... For reference, the codes for training are mainly based on the [Monodepth2](https://github.com/nianticlabs/monodepth2), and we mainly change the architecture of PoseNet and the calculation of training loss. I will find some time to clean up the codes in the `dev` branch for easy use.
